jasper.proto

```diff
@@ -94,6 +95,36 @@ message BuildloggerV2Options {
   BaseOptions base = 2;
 }
 
+message BuildloggerV3Info {
+  string project = 1;
+  string version = 2;
+  string variant = 3;
+  string task_name = 4;
+  string task_id = 5;
+  int32 execution = 6;
+  string test_name = 7;
+  int32 trial = 8;
+  string process_name = 9;
+  LogFormat format = 10;
+  repeated string tags = 11;
+  map<string, string> args = 12;
+  bool mainline = 13;
+  int64 max_buffer_size = 14;
+  int64 flush_interval = 15;
+  bool disable_new_line_check = 16;
+  string rpc_address = 17;
+  bool insecure = 18;
+  string ca_file = 19;
+  string cert_file = 20;
+  string key_file = 21;
+}
+
+message BuildloggerV3Options {
+  BuildloggerV3Info buildloggerv3 = 1;
```

Comment:
why split between info and options? 

maybe split the first 9 fields out?

Reply:
Don't have much flexibility here... this has to copy exactly whats [here](https://github.com/evergreen-ci/timber/blob/master/logger_producer.go#L11-L15) in timber so it can marshal correctly.